README.md

```diff
@@ -100,10 +100,10 @@ If you'd like to learn more about running `jest` through the command line, take
 
 [Babel](http://babeljs.io/) is automatically handled by Jest using `babel-jest`. You don't need install anything extra for using Babel.
 
-> Note: If you are using a babel version 7 then you need to install `babel-core@^7.0.0-0` and `@babel/core` with the following command:
+> Note: If you are using Babel version 7, then you need to install `babel-core@^7.0.0-bridge.0` and `@babel/core` with the following command:
 >
 > ```bash
-> yarn add --dev 'babel-core@^7.0.0-0' @babel/core
```

Comment:
Should keep the quotes, see https://github.com/facebook/jest/pull/6746

Reply:
The quotes throw an error.
![image](https://user-images.githubusercontent.com/343837/43156933-11d5b30a-8f41-11e8-9c06-40575b9ccc10.png)

Changing it to match the getting started page (re: #6746) just means the getting started page is wrong too.

Reply:
What I got from that Issue link is someone saying "There should be quotes to match the docs," as opposed to "There should be quotes in order for this to work." Let me know if the latter is the case @SimenB. I did not notice evidence that the quotes are needed for this to work; however, I do have evidence that the quotes must not exist for this to work.

Reply:
Any update on this? I do not agree with your requested changes @SimenB. I believe leaving the quotes breaks the command, as per the screenshots.

Reply:
@SimenB installing without quotes works just fine for me on latest Yarn.

Reply:
Both works for me, but since we got a PR I guess it doesn't work for everybody. Is it a Windows thing?

Reply:
@SimenB I am on Windows, so that may be the issue.